src/components/AddItemModal.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import DefaultModal from './DefaultModal';
+import Logo from '../molecules/Logo';
+import TextInput from './TextInput';
+
+const Body = styled.div`
+	height: 20em;
+`;
+
+const Desc = styled.p`
+	color: black;
```

Comment:
For some reason it's giving me white as default?

Reply:
Okay we should nuke the app.css, but I am wondering why that class is being applied to your p tag anyways?

I feel like this should affect your p tag since it doesn't actually have a class attached, but we can literally delete the app.css file, and I want to say do it in this ticket despite that being bad practice